codalab/model/bundle_model.py

```diff
@@ -539,113 +543,114 @@ def make_condition(key, field, value):
                     )
 
                 subclause = None
+                aliased_bundle_metadata = aliased(cl_bundle_metadata)
                 if key == '.before':
-                    subclause = cl_bundle_metadata.c.metadata_value <= int(
+                    subclause = aliased_bundle_metadata.c.metadata_value <= int(
                         target_datetime.timestamp()
                     )
                 if key == '.after':
-                    subclause = cl_bundle_metadata.c.metadata_value >= int(
+                    subclause = aliased_bundle_metadata.c.metadata_value >= int(
                         target_datetime.timestamp()
                     )
-
-                clause = cl_bundle.c.uuid.in_(
-                    alias(
-                        select([cl_bundle_metadata.c.bundle_uuid]).where(
-                            and_(cl_bundle_metadata.c.metadata_key == 'created', subclause)
-                        )
-                    )
+                add_join(
+                    aliased_bundle_metadata,
+                    cl_bundle.c.uuid == aliased_bundle_metadata.c.bundle_uuid,
                 )
+                where_clause = and_(aliased_bundle_metadata.c.metadata_key == 'created', subclause)
             elif key == 'uuid_name':  # Search uuid and name by default
-                clause = []
-                clause.append(cl_bundle.c.uuid.like('%' + value + '%'))
-                clause.append(
-                    cl_bundle.c.uuid.in_(
-                        alias(
-                            select([cl_bundle_metadata.c.bundle_uuid]).where(
-                                and_(
-                                    cl_bundle_metadata.c.metadata_key == 'name',
-                                    cl_bundle_metadata.c.metadata_value.like('%' + value + '%'),
-                                )
-                            )
-                        )
+                aliased_bundle_metadata = aliased(cl_bundle_metadata)
+                add_join(
+                    aliased_bundle_metadata,
+                    cl_bundle.c.uuid == aliased_bundle_metadata.c.bundle_uuid,
+                )
+
+                where_clause = []
```

Comment:
To be more accurate, I'd call this `where_clause_disjuncts`

Reply:
Thanks! (I think they're actually conjuncts in this case since we use and_ to connect them in line 623, but please correct me if I'm wrong!)